README.md

```diff
@@ -45,14 +47,6 @@ Create a secret called `github-runner-app` by executing the following command in
 kubectl create secret generic github-runner-app --from-literal=GITHUB_APP_INTEGRATION_ID=<app_id> --from-file=GITHUB_APP_PRIVATE_KEY=<private_key>
 ```
 
-Finally define the following on the operator deployment:
```

Comment:
I don't think this should be removed as it's describing how it actually works (although helm abstracts that)

Reply:
This value is already set in the [deployment.yaml](https://github.com/evryfs/helm-charts/blob/master/charts/github-actions-runner-operator/templates/deployment.yaml) if you were to download it manually and apply it.
For the approach mentioned in this section of the readme, this value both doesn't need to be set and cannot be set, without downloading the helm chart and deploying it manually.

Reply:
When trying to follow these instructions, I got held up by not being able to find how or where to define that `secretRef`. I then found that it didn't need to be set at all, so for clarity, it should not be in the readme

Reply:
@davidkarlsen, what should happen to resolve this code review so it can get merged into main?